sdk/core/core/inc/az_context.h

```diff
@@ -26,16 +26,20 @@ typedef struct az_context az_context;
  */
 struct az_context
 {
+  /// @internal
```

Comment:
looks like this is not a valid Doxygen command
```
 error: unknown command tag name [-Werror,-Wdocumentation-unknown-command]
/// @internal
    ^~~~~~~~~
```

Reply:
internal would only hide the documentation that is written after it
like
```c
/** @brief Some comment that is visible in docs
    @internal Some info not visible in docs due to internal!  applies to all this line
    This is now visible in docs
    @internal for
    multi line internal 
    we use the endofinternal
    @endofinternal
```

Reply:
It should be possible to get rid of the -Wdocumentation errors, but I am afraid you're right that internal/endinternal do only apply to documentation comment paragraphs and not whole regions of the file.